src/lib/features/listening-history/model/specs/store.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { addToListeningHistory, listeningHistory } from '../store';
+import { get } from 'svelte/store';
+import type { Episode } from '$lib/shared/api';
+
+const exampleEpisode: Episode = {
+  id: 1,
+  episodeNumber: 1,
+  podcastID: 1,
+  title: 'Episode 1',
+  audioUrl: 'http://example.com',
+  duration: 5,
+};
+
+const expectedHistory: number[] = [exampleEpisode.id];
+
+it('adds an episode to the history', () => {
```

Comment:
suggestion (non-blocking): I think this test suite should also test the subscriptions — that the history is fetched on login and that playing an episode adds it to history. However, this expands from unit tests into integration tests slightly, so do it if you see fit, we might cover it in large-scale E2E tests later anyway